library.py

```diff
@@ -39,7 +39,7 @@
 if atstart_file is None:
     sys.stderr.write("Error: Please specify property `atstart_file = myproject.atstart`.\nThe file is expected to be in the project source directory.\n")
     env.Exit(1)
-input_filename = os.path.join(env['PROJECTSRC_DIR'], atstart_file)
+input_filename = os.path.join(env['PROJECT_DIR'], atstart_file)
```

Comment:
Please use `git mv` to move the location of the .atstart file in the other example, so that it works, too.

Reply:
Understood. I'm ambivalent and appreciate your nudge. If someone wants to organize it differently, putting atstart files in project subdir, they can add a path prefix to the parameter.